src/modules/projects/components/tables/ProjectReferralPostingsTable.tsx

```diff
@@ -63,6 +52,16 @@ interface Props {
 export const ProjectReferralPostingsTable: React.FC<Props> = ({
   projectId,
 }) => {
+  const rowLinkTo = useCallback(
```

Comment:
Can you add `linkTreatment: true,` to the HoH name column definition? Without it, there is no way to tab through the rows, because we use the "linked" cell  as the tab index (even though all cells in the row are clickable). The link treatment also indicates to the user visually that the row is clickable.

Reply:
This should be done for the referral request table as well

Reply:
One thing with the referral request table is that it doesn't have a rowLinkTo. I added the linkTarget to the cancel button since it's the only clickable element there but that may not be correct

Reply:
Oh right, I forgot that the referral request table rows aren't clickable. All looks correct now though. The void buttons are tabbable, as are the HoH Name cells on the referral table.